### DISCUSSION TITLE
20220802

### DISCUSSION THREAD

**Discussion by indygreg** [indygreg/python-build-standalone#137] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/indygreg/python-build-standalone/releases/tag/20220802'>20220802</a>.</em>

**Comment by polkovnikov** [comment]
@indygreg Hi! There was [my Issue](https://github.com/indygreg/python-build-standalone/issues/92) that I posted exactly 1 year ago.

It suggests to incorporate support for isolated sub-interpreters. This is experimental feature, which should be enabled separately, it is done through

`./configure --with-experimental-isolated-subinterpreters`

when compiling CPython. Or by passing `#define EXPERIMENTAL_ISOLATED_SUBINTERPRETERS 1` when doing same compilation.

My question is how to pass both these ways to build script? When I call `python-build-standalone/cpython-windows/build.py` can I pass these flags somehow?

I'm interested in question how can I re-build CPython using your tools so that this feature is enabled?

**Reply by indygreg** [reply]
Let's discuss this on issue #92.
